tailwind.config.js

```diff
@@ -2,7 +2,15 @@
 module.exports = {
   content: ["./*.{html,js}"],
   theme: {
-    extend: {},
+    extend: {
+      colors:{
+        ourPrimaryGreen: "#00672E",
+        bgPrimary: "EFECEA"
+      }
+    },
+    fontFamily: {
+      body: ["Outfit"]
```

Comment:
It's a good practice to add the backup font family, which will be applied just in case the font isn't loaded.